apps/web-giddh/src/app/reports/components/columnar-report-table-component/columnar.report.table.component.html

```diff
@@ -18,27 +18,27 @@
                         <!--Parent  Name -->
                         <td data-title="Parent Group"> {{ item.groupName}} </td>
                         <!--Opening balance  -->
-                        <td class="text-center" *ngIf="(isShowClosingOpeningBalance$ | async)">
+                        <td data-title="Opening Balance" class="text-center" *ngIf="(isShowClosingOpeningBalance$ | async)">
                             <span *ngIf="isCrDrChecked">{{(columnarReportResponse?.openingBalance[item.accountUniqueName]?.type ==='DEBIT' && columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount >0)? '-': ''}} </span> {{ columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount}}
                             <span *ngIf="!isCrDrChecked && columnarReportResponse?.openingBalance[item.accountUniqueName]?.amount >0">{{columnarReportResponse?.openingBalance[item.accountUniqueName]?.type ==='DEBIT'? 'Dr': 'Cr'}}</span>
                         </td>
                         <!--Closing balance  -->
-                        <td class="text-center" *ngIf="(isShowClosingOpeningBalance$ | async)">
+                        <td data-title="Closing Balance" class="text-center" *ngIf="(isShowClosingOpeningBalance$ | async)">
                             <span *ngIf="isCrDrChecked && columnarReportResponse?.closingBalance[item.accountUniqueName]?.amount>0"> {{(columnarReportResponse?.closingBalance[item.accountUniqueName]?.type ==='DEBIT')?'-':''}} </span>{{columnarReportResponse?.closingBalance[item.accountUniqueName]?.amount}}
                             <span *ngIf="!isCrDrChecked && columnarReportResponse?.closingBalance[item.accountUniqueName]?.amount>0">
                             {{columnarReportResponse?.closingBalance[item.accountUniqueName]?.type
                             ==='DEBIT'?'Dr':'Cr'}} </span>
                         </td>
                         <!--Grand  Total -->
-                        <td class="text-center">
+                        <td data-title="Grand  Total" class="text-center">
                             <span *ngIf="isCrDrChecked && columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount > 0"> {{(columnarReportResponse?.grandTotal[item.accountUniqueName]?.type ==='DEBIT')?'-':''}}</span> {{columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount}}
                             <span *ngIf="!isCrDrChecked && columnarReportResponse?.grandTotal[item.accountUniqueName]?.amount > 0">
                              {{columnarReportResponse?.grandTotal[item.accountUniqueName]?.type
                             ==='DEBIT'?'Dr':'Cr'}}</span>
                         </td>
                         <ng-container *ngFor="let months of columnarReportResponse.data;let inx2 = index;">
                             <!-- months data -->
-                            <td class="text-center">
+                            <td class="text-center" attr.data-title={{monthName[inx2]}}>
```

Comment:
Please use correct binding
```suggestion
                            <td class="text-center" [attr.data-title]={{monthName[inx2]}}>
```

Reply:
Done